ticdc/manage-ticdc.md

```diff
@@ -588,8 +588,8 @@ curl -X POST -d '"debug"' http://127.0.0.1:8301/admin/log
 # 该配置会同时影响 filter 和 sink 相关配置，默认为 true
 case-sensitive = true
 
-# 是否输出 old value，从 v4.0.5 开始支持
-enable-old-value = false
```

Comment:
Please help to confirm the version barrier, where `enable-old-value` is changed from false to true @liuzix

Reply:
https://github.com/pingcap/ticdc/commit/6c633b5aa4722a170309be20be39f132e15f635a  I think is 5.0.